### PR TITLE
PR template

### PR DESCRIPTION
**Pull request by PeterNgTr**
Propose a PR template to give us a nicer overview of the PR.

### CODE REVIEW

File: PR_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## Motivation/Description of the PR
+
+- A link to the issue (for bugfixes) or a description of the new feature.
```
Comment (DavertMik) [inline PR_TEMPLATE.md]:
```suggestion
- Description of this PR, which problem does it solve
- A link to the corresponding issue (if applicable).
```

File: PR_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- [ ] Bug fix
+- [ ] Documentation change/update
+- [ ] Hot fix
+
+## Checklist:
+
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] Tests have been added
+- [ ] Documentation has been added
```
Comment (DavertMik) [inline PR_TEMPLATE.md]: Maybe to mention above or below this section on how to update documentation...

**Comment by Vorobeyko** [comment]
Maybe move all templates to .github folder?

**Comment by Vorobeyko** [comment]
What about the file name? 
Should this be the name PULL_REQUEST_TEMPLATE? Or is PR_TEMPLATE also possible?

File: PR_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+- [ ] Hot fix
+- [ ] Markdown files fix - not related to source code
+
+## Checklist:
+
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] Tests have been added
+- [ ] Documentation has been added (Run `robo docs`)
+- [ ] Lint checking (Run `npm run lint`)
```
Comment (Vorobeyko) [inline PR_TEMPLATE.md]: Lint and the test running on each push and in the pipeline, why is this added to checklist?

Comment (PeterNgTr) [reply PR_TEMPLATE.md]: with some PRs, I still see the lint issues, so better to add this so they could fix it before committing.

**Comment by PeterNgTr** [comment]
I moved templates to `.github` folder as described here https://github.blog/2016-02-17-issue-and-pull-request-templates/

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+- Description of this PR, which problem it solves
+- A link to the corresponding issue (if applicable).
+
+## Type of change
+
+- [ ] Breaking changes
+- [ ] New functionnality
+- [ ] Bug fix
+- [ ] Documentation change/update
+- [ ] Hot fix
```
Comment (elukoyanov) [inline .github/PULL_REQUEST_TEMPLATE.md]: Not sure, that  "hot fix" should be here.

It's the same bug fix, but only urgent

Comment (PeterNgTr) [reply .github/PULL_REQUEST_TEMPLATE.md]: Yeah, it’s the point. Make the release to hot fix it. Bug fix can wait.
